core/src/com/bot4s/telegram/future/Polling.scala

```diff
@@ -12,7 +12,7 @@ trait Polling extends BasePolling[Future] with BotExecutionContext with StrictLo
 
   private type OffsetUpdates = (Option[Long], Seq[ParsedUpdate], User)
 
-  @volatile private var polling: Future[Unit] = _
+  @volatile private var polling: Future[Unit] = Future.unit
```

Comment:
This is not working:
```
./mill -i 'examples.jvm[2.13.16].console'

scala> new RandomBot("BOT KEY").run()
java.lang.RuntimeException: Bot is already running
  at com.bot4s.telegram.future.Polling.run(Polling.scala:69)
  at com.bot4s.telegram.future.Polling.run$(Polling.scala:67)
  at RandomBot.run(RandomBot.scala:13)
  ... 30 elided
```

This should probably be `@volatile private var polling: Future[Unit] = null`

If we want to keep the behavior as close as possible to the original. We might want to change that in the future, but this PR already makes a lot of changes on the syntax, I would try to avoid changing anything related to the feature.

Reply:
Done 🤝